src/assets/TestFramework/Deployment.cs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+using System;
+
+namespace Azure.ResourceManager.TestFramework
+{
+    /// <summary> A Class representing a Deployment along with the instance operations that can be performed on it. </summary>
+    public partial class Deployment : ArmResource
```

Comment:
In the latest version of azure.resourcemanager.resources, this resource has been renamed to `ArmDeploymentResource` according to comments from arch board. We might need to change this accordingly
See here: https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/resources/Azure.ResourceManager.Resources/src/Generated/ArmDeploymentResource.cs